search_gov_crawler/elasticsearch/i14y_helper.py

```diff
@@ -20,18 +26,39 @@
 }
 # fmt: on
 
-def null_date(article_date):
+load_dotenv()
+logging.basicConfig(level=os.environ.get("SCRAPY_LOG_LEVEL", "INFO"))
+logging.getLogger().handlers[0].setFormatter(JsonFormatter(fmt=LOG_FMT))
+logger = logging.getLogger("search_gov_crawler.i14y_helper")
+
+
+def parse_date_safely(date_value: Any) -> str | None:
     """
     Convert falsey date values (like an empty string) to None,
-    which will yield a null value in elasticsearch
+    which will yield a null value in elasticsearch.
 
     Args:
-        article_date (str | Any): Date value that needs to fallback to null
+        date_value (str | Any): Date value that needs to fallback to null
 
     Returns:
-        str: Either the original date, or NoneValue
+        str: Either the date in isoformat (YYYY-MM-DD'T'HH:MM:SS), or None
     """
-    return article_date or None
+    datetime_format = "%Y-%m-%dT%H:%M:%S"
+
+    if date_value is None or date_value == "":
+        return None
+
+    if isinstance(date_value, datetime):
+        return date_value.strftime(datetime_format)
```

Comment:
I don't think we should introduce or try to mimic ISO standard format. The datetime class has it builtin, so it would look something like this:
```suggestion
        return f"{date_value.isoformat()}Z"
```